config/routes.rb

```diff
@@ -270,6 +270,7 @@ def external_redirect(path_prefix, target)
             post :upload_zip, on: :collection
             get :set_titles, on: :member
           end
+          resources :images
```

Comment:
generally when adding endpoints it's good to just add the ones you need as you need them. It might be that you end up needing new/create/edit/update etc., but for now i'd just add 

```
resources :images, only: %i[index]
```